.pre-commit-config.yaml

```diff
@@ -11,6 +11,11 @@ repos:
       - id: check-yaml
       - id: check-added-large-files
       - id: check-toml
+  - repo: https://github.com/python-poetry/poetry
+    rev: 1.8.0
+    hooks:
+      - id: poetry-check
+      - id: poetry-lock
```

Comment:
I'm in favor of including `poetry-check`, but I'm uncertain whether or not we want to be including `poetry-lock` as well. Because `poetry-lock` checks to see whether dependencies are the "most up to date", it could run differently on a user's machine vs later when the same commit gets pushed to github. I think instead we should rely on dependabot for package updates.

Reply:
Alternately, we could add the `--no-update` argument to the `poetry-lock` check